src/core/tests/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ ov_add_test_target(
             Threads::Threads
             openvino::conditional_compilation
             openvino::runtime::dev
+            openvino_core_obj
```

Comment:
I think this shouldn't be required the linking should work in same way it looks like something is not exported from openvino correctly

Reply:
yes. I put OPENVINO_API in wrong place